src/libstore/worker-protocol.hh

```diff
@@ -10,6 +10,9 @@ namespace nix {
 #define WORKER_MAGIC_2 0x6478696f
 
 #define PROTOCOL_VERSION (1 << 8 | 34)
+#define PROTOCOL_MAJOR(x) (((x) & 0xff00) >> 8)
+#define PROTOCOL_MINOR(x) ((x) & 0x00ff)
```

Comment:
it seems like it would make sense to give the different major/minor macros (or functions if the other advise is followed) differing names as this also makes it easier to grep for their usage etc.